app/services/explorer.js

```diff
@@ -852,6 +876,14 @@ export default Ember.Service.extend({
             .then(function(PromiseArray) {
                 let cluster = PromiseArray[0].value;
 
+                // Create search-schemas from index references
+                //  and set the schema/index association
+                cluster.get('searchIndexes').forEach(function(index) {
+                   let schema = self.createSchema(index.get('schemaRef'), cluster, store);
+
+                   index.set('schema', schema);
+                });
```

Comment:
Since this is currently the only way to get the schema's until https://github.com/basho-labs/riak_explorer/issues/89 is done, this seemed like the most logical approach.